Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/ProtoFormatConverterExtensions.kt

```diff
@@ -33,9 +33,9 @@ object ProtoFormatConverterExtensions {
         this.sortedWith(compareBy { it.rollingStartIntervalNumber }).asReversed().take(MAXIMUM_KEYS)
 
     fun List<TemporaryExposureKey>.transformKeyHistoryToExternalFormat() =
-        this.sortedWith(compareBy { it.rollingStartIntervalNumber })
+        this.sortedWith(compareBy { it.rollingStartIntervalNumber }).asReversed()
```

Comment:
@mh- next to the asReversed option I see the other option to use "compareByDescending" instead of "compareBy". This option would result into better readability of code.

Do you see anything speaking against the usage of "compareByDescending"? (for sure we remove as Reversed in all the places)

Reply:
@ChristopherSchmitz `compareByDescending` is probably better, if it makes the code more readable. I don't read Kotlin a lot and indeed had a hard time understanding the code. So I just copied the pattern from [.limitKeyCount()](https://github.com/corona-warn-app/cwa-app-android/blob/c2e5bd49093193060dded4d1545f783a498a96da/Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/ProtoFormatConverterExtensions.kt#L33) and wrote a test case to see what it's actually doing. 
The test case assumes that the keys are ordered chronologically, but if that's not required for upload to the server, we could also move the 2nd sorting to the test.
Not sure how you prefer to proceed? Do you want to edit the PR, or should I do it?

Reply:
@mh since we use on iOS a desc order when submitting to the server I would keep the behaviour consistent.
If you like to adjust your PR this would be wonderful. We plan to close and merge this in the next 2 hours :-)

Reply:
@ChristopherSchmitz done.